### DISCUSSION TITLE
Nozzle size

### DISCUSSION THREAD

**Discussion by JaVelin-X** [5axes/Calibration-Shapes#88] [Q&A]
How do big nozzles impact how to use these test parts? should some of them be scaled up for bigger nozzles? I can't find it now but I thought I read most of the parts are for .4mm nozzles. I have a print head with a 1.2mm nozzle I use a lot but one printer gives me a better result than the other with the same head so I wanted to investigate that

**Comment by 5axes** [comment]
All parts have been designed for the use of a 0.4 Nozzle, but most of them and also some tests realized by the plugin offer the possibility to use a nozzle up to 0.8 mm. For a nozzle of 1.0 or 1.2, yes I recommend to scale up your part by almost a factor 2.

**Reply by aforww** [reply]
I'd love to see the post-processing have an option for nozzle diameter.  As an example, the temp tower assumes a layer height of .16 but no way am I doing those layer heights with a .6 nozzle much less a .8 on a day-to-day print.  Not saying the prints can't be done, after some maths to change the heights based on a .2 height but it sure would be nice for it to do it on its own.  Great plugin otherwise.

**Reply by 5axes** [reply]
Well this request make sens but it's not so easy to offer the right solution. We have different  solution to do that .

A) Try to use a unique model design and set some automaticaly some parameter in cura to avoid "Newbie" question : The actual solution
      It's difficult to have just one model , we could use a bigger model but in this case for a 0.4 or 0.2 nozzle the printing time will be too long

B) According to the nozzle size use different models
      It could be confusing for the user to get different result and for the support it's also a problem if you don't know what kind of model are used to get the result.
     Solution easy to integrate but the definition of the model according to the Nozzle Size is not so easy to do, and must be validate by several users.
